fili-core/src/test/java/com/yahoo/bard/webservice/models/druid/client/impl/TestDruidWebService.java

```diff
@@ -114,7 +115,12 @@ public void postDruidQuery(
         }
 
         // Set the response to use based on the type of the query we're processing
-        switch (lastQuery.getQueryType()) {
+        if (!(lastQuery.getQueryType() instanceof DefaultQueryType)) {
+            throw new IllegalArgumentException("Illegal query type : " + lastQuery.getQueryType());
```

Comment:
We should add a note (likely to the changelog) that this test extension point doesn't support new types of queries.